### DISCUSSION TITLE
SASS Output

### DISCUSSION THREAD

**Discussion by pawanmsr** [pygments/pygments#2650] [Q&A] [answered]
Hi,

I hope you are having a wonderful time. I was wondering if `pygmentize` command outputs stylesheet as *sass* as well (along with the usual *css*).

```shell
# Something along the following lines but for sass instead of css.
pygmentize -f html -S $style -a . . . 

```

**Comment by jeanas** [comment, marked as answer]
A late reply, but I'm not sure why you want Sass "instead of" CSS given that all valid CSS is also valid Sass?

**Reply by jeanas** [reply]
Hm. The front page of https://sass-lang.com says "Sass is completely compatible with all versions of CSS." but https://sass-lang.com/guide/ says "Sass has two syntaxes! The SCSS syntax (.scss) is used most commonly. It’s a superset of CSS, which means all valid CSS is also valid SCSS. The indented syntax (.sass) is more unusual: it uses indentation rather than curly braces to nest statements, and newlines instead of semicolons to separate them." (Obviously, I'm not a Web designer.) So you want the stylesheets in that "unusual" syntax?

**Reply by pawanmsr** [reply]
Hi, thanks a lot. The CSS worked just fine. You may delete/close the post. Thank you very much for replying.

Although, if you agree, may I please add additional functions/scripts (in pygments) to convert to SCSS and SASS as well (in a single PR); to keep it up to date and just in case someone else wants a SCSS or [SASS](https://sass-lang.com/) output (in future)?

**Reply by jeanas** [reply]
Yes, a PR to add an `HtmlFormatter` subclass producing SASS would be welcome.
